### PR TITLE
Remove Bash script, update action

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

File: .github/update.sh [hunk 1]
```diff
@@ -8,15 +8,15 @@
 LUPDATE=$(which lupdate)
 LCONVERT=$(which lconvert)
 
-LUPDATE vpn/src/src.pro -ts
+$LUPDATE "$1/src/src.pro" -ts
```
Comment (flodolo) [inline .github/update.sh:11]: I think this is the right approach, since the vpn path is passed to the script.

File: .github/update.sh [hunk 2]
```diff
@@ -8,15 +8,15 @@
...
 
 for FILE in $1/translations/*; do
-    OUT_FILE="$2/$(basename '$FILE' .ts).xliff"
+    OUT_FILE="$2/$(basename $FILE .ts).xliff"
```
Comment (flodolo) [inline .github/update.sh:14]: With the quote, it remains `$FILE.xliff` :-(

File: .github/update.sh [hunk 3]
```diff
@@ -8,15 +8,15 @@
...
     if [ -f $OUT_FILE ]; then
         echo "Updating $OUT_FILE"
-        LCONVERT -i "$FILE" -i $OUT_FILE -o $OUT_FILE
+        $LCONVERT -if xlf -i "$FILE" -i $OUT_FILE -of xlf -o $OUT_FILE
```
Comment (flodolo) [inline .github/update.sh:17]: Not completely sure about the input/output formats here, based on your changes to the Python file.

**Review by strseb — CHANGES_REQUESTED** [review]
So sorry you had to fiddle with this. 
We can safely delete the .sh, its not used anymore 🙈

**Comment by flodolo** [comment]
No problem, removing the file and updating the action then ;-)

**Review by strseb — APPROVED** [review]
Great, thanks! :)

**Pull request merged**